k9mail/src/main/java/com/fsck/k9/ui/compose/QuotedMessagePresenter.java

```diff
@@ -173,7 +174,8 @@ public void onSaveInstanceState(Bundle outState) {
     public void onRestoreInstanceState(Bundle savedInstanceState) {
         quotedHtmlContent = (InsertableHtmlContent) savedInstanceState.getSerializable(STATE_KEY_HTML_QUOTE);
         if (quotedHtmlContent != null && quotedHtmlContent.getQuotedContent() != null) {
-            view.setQuotedHtml(quotedHtmlContent.getQuotedContent());
+            // we don't have the part here, but inline-displayed images are cached by the webview
+            view.setQuotedHtml(quotedHtmlContent.getQuotedContent(), null);
```

Comment:
this is hacky because quoted messages had no proper concept of how to load inline attachments before at all and relied on cached data entirely (try forwarding from context menu).

this should be easy to get right once the decoded message state is stored by a MessageLoaderHelper (from #1299), and QuotedMessagePresenter doesn't have to worry about it anymore